qemu/tests/cfg/block_commit.cfg

```diff
@@ -0,0 +1,45 @@
+# Below case works stablly on QMP monitor, but have no more test with Human
+# monitor, so suggest to use QMP monitor as default qemu monitor run below test
+#
+- block_commit:
+    no raw qed vmdk
+    backup_image_before_testing = yes
+    restore_image_after_testing = yes
+    wait_finished = yes
+    source_image = image1
+    default_speed_image1 = 0
+    snapshot_chain = "images/sn1"
+    # If image size increase or limited committing please increase
+    # wait_timeout to avoid wait for committing finished timeout;
+    wait_timeout = 3900
+    snapshot_format = qcow2
+    kill_vm = yes
+    alive_check_cmd = dir
+    tmp_dir = /tmp
+    variants:
+        - with_stress:
```

Comment:
There so many place has such kind of duplicate configurations,
Here is an idea, let's add below code in guest-os/windows.cfg
```
with_stress:
         install_path = "C:\Program Files (x86)\JAM Software\HeavyLoad"
         config_cmd = 'setx -m path "%PATH%;${install_path};"'
         install_cmd = "start /wait DRIVE:\HeavyLoadSetup.exe /verysilent"
         check_cmd = 'tasklist | findstr /I  "heavyload.exe"'
         stop_cmd = "taskkill /T /F /IM heavyload.exe"
         x86_64:
             ...
         i386,i686:
             ...
```
For Linux guest, we can move common configuration to guest-os/Linux.cfg too.

do it when just need to named our stress related test to xxx..with_stress, then we are no need to copy this kind of configuration to our test.cfg again and again.